cmd/fossa/cmd/analyze/analyze.go

```diff
@@ -45,7 +50,26 @@ func Run(ctx *cli.Context) error {
 		log.Logger.Fatal("No modules specified.")
 	}
 
-	var analyzed []module.Module
+	analyzed, err := AnalyzeModules(modules)
+	if err != nil {
+		log.Logger.Fatalf("Could not analyze modules: %s", err.Error())
+		return err
+	}
+
+	normalized, err := fossa.Normalize(analyzed)
+	if err != nil {
+		log.Logger.Fatalf("Could not normalize output: %s", err.Error())
+		return err
+	}
+
+	if ctx.String(Output) != "" || ctx.String(Template) != "" {
+		return outputAnalysis(ctx.String(Output), ctx.String(Template), normalized)
+	}
+
+	return uploadAnalysis(normalized)
+}
+
+func AnalyzeModules(modules []module.Module) (analyzed []module.Module, err error) {
```

Comment:
func name will be used as analyze.AnalyzeModules by other packages, and that stutters; consider calling this Modules